builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -143,4 +146,15 @@ private boolean isValidToken(final String token) {
         }
     }
 
+    
+    private boolean isRequestFromGwtFrame(String requestURI) {
+        URI uri;
+        try {
+            uri = new URI(requestURI);
+            return "/api/java/javadoc/find".equals(uri.getPath()) // Java Quick Documentation popup
```

Comment:
> Do we really need to create URI object?
> Previously we used "endwith()" approach

There are parameters in the expected URIs, and using `String.contains()` didn't seem precise enough to me.